network-mux/network-mux.cabal

```diff
@@ -161,3 +161,36 @@ executable mux-demo
                        -Wpartial-fields
                        -Widentities
                        -Wredundant-constraints
+
+executable cardano-ping
+  hs-source-dirs:      demo, test
+  main-is:             cardano-ping.hs
+  other-modules:       Test.Mux.ReqResp
+  build-depends:       base,
+                       directory,
+                       network-mux,
+                       io-sim-classes,
+                       contra-tracer,
+                       stm,
```

Comment:
What do you need from `stm` package that's not in `io-sim-classes`?

Reply:
Nothing, will remove.